packages/dds/ink/package.json

```diff
@@ -60,6 +60,7 @@
     "@fluidframework/datastore-definitions": "^0.55.0",
     "@fluidframework/driver-utils": "^0.55.0",
     "@fluidframework/protocol-definitions": "^0.1026.0",
+    "@fluidframework/runtime-utils": "^0.55.0",
```

Comment:
why did we choose runtime-utils for this? I'm wondering if datastore-definitions would be better

Reply:
or if its really only needed by dds then shared-object-base

Reply:
it seems like even the implementation in runtime-utils can move to share object

Reply:
I put it in runtime-utils to go with FluidSerializer. shared-object-base makes sense.